bindings/python/pymongoarrow/types.py

```diff
@@ -288,7 +288,7 @@ def _normalize_typeid(typeid, field_name):
             fields.append((sub_field_name, _normalize_typeid(sub_typeid, sub_field_name)))
         return struct(fields)
     if isinstance(typeid, list):
-        return list_(_normalize_typeid(type(typeid[0]), "0"))
+        return list_(_normalize_typeid(typeid[0], "0"))
```

Comment:
What happens if the provided list has >1 value in it? Should we error or silently ignore it?

Reply:
Good point, I would say we could raise with a message that the struct should be provided.

Reply:
That sounds reasonable to me. A ValueError would be appropriate.

Reply:
Done